src/text.md

```diff
@@ -279,22 +286,31 @@ We know that this is a hard line to take, and it pulls hard for the security sid
 ## Smart Speakers
 If you have a smart speaker, **throw it out**.
 
-Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we judge the potential tradeoffs to be too high. We judge it impossible to confidently and securely host a smart speaker in your house — so don't.
+Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we think the potential tradeoffs are too high. We possible to confidently and securely host a smart speaker in your house — so don't.
```

Comment:
```suggestion
Smart speakers like Alexa and Google Home sit in your house and can listen to everything you and your family say, then send those recordings back to Amazon and Google, where you have no control over who can listen to them or give them to someone else. Even with the privacy policies that accompany these devices, even with the quality-of-life improvement, *even* with the Three Dumb Router approach, we think the potential tradeoffs are too high. We are of the opinion that it is impossible to confidently and securely host a smart speaker in your house — so don't.
```

Reply:
I have no idea what the heck happened here.